collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -142,9 +142,9 @@ public void playVideo() {
 
             Intent i = new Intent("android.intent.action.VIEW");
             i.setDataAndType(Uri.fromFile(videoFile), "video/*");
-            try {
+            if (i.resolveActivity(getContext().getPackageManager()) != null) {
```

Comment:
This PR contains some old commits. Because of that, old changes are also coming up. You check before branching whether it is from master or not.